contrib/docker-integration/run_multiversion.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+# Run the integration tests with multiple versions of the Docker engine
+
+set -e
+
+# Don't use /tmp because this isn't available in boot2docker
+tmpdir_template="`pwd`/docker-versions.XXXXX"
+tmpdir=`mktemp -d "$tmpdir_template"`
+trap "rm -rf $tmpdir" EXIT
+
+# If DOCKER_VOLUME is unset, create a temporary directory to cache containers
+# between runs
+# Only do this on Linux, because using /var/lib/docker from a host volume seems
+# problematic with boot2docker.
+if [ "$DOCKER_VOLUME" = "" -a `uname` = "Linux" ]; then
```

Comment:
The idea is to share images between the different invocations of run.sh that happen below.